Emby.Server.Implementations/LiveTv/TunerHosts/BaseTunerHost.cs

```diff
@@ -83,9 +83,8 @@ public async Task<List<ChannelInfo>> GetChannels(bool enableCache, CancellationT
                 try
                 {
                     var channels = await GetChannels(host, enableCache, cancellationToken).ConfigureAwait(false);
-                    var newChannels = channels.Where(i => !list.Any(l => string.Equals(i.Id, l.Id, StringComparison.OrdinalIgnoreCase))).ToList();
```

Comment:
Why?

Reply:
List appears to be empty so this line was doing nothing of benefit. I assume it's meant to be used to determine what channels are new and to only pull those in. I plan on implementing a working version of that behavior in the future.

Reply:
It's in a for-loop though. If multiple tuner hosts contain the same channel there'll be an overlap

Reply:
Good catch completely overlooked that. I'll revert.

Reply:
You can remove the `.ToList` though and save some allocations.